modules/web/src/com/haulmont/sampler/web/ui/components/popupbutton/customlayout/messages.properties

```diff
@@ -0,0 +1,5 @@
+popupButtonCaption = Settings
+saveButtonCaption = Save
+resetButtonCaption = Reset
```

Comment:
Let's call it **Cancel**. Usually, **reset** close neither the dialog window nor the popup, it's just clear the fields. In our case, this is the **Cancel** button.